app/views/auth_sources/_welcome.html.erb

```diff
@@ -1,19 +1,6 @@
-<div class="blank-slate-pf">
-  <div class="blank-slate-pf-icon">
-    <%= icon_text("users", "", :kind => "fa") %>
-  </div>
-  <h1><%= _('Authentication Sources') %></h1>
-  <p>
-    <%= _("Foreman can use LDAP based service for user information and authentication.") %><br>
-    <%= _("The authentication process currently requires an LDAP provider, such as <em>FreeIPA</em>,
-        <em>OpenLDAP</em> or <em>Microsoft's Active Directory</em>.").html_safe %>
-  </p>
-  <p><%= link_to _('Learn more about LDAP authentication in the documentation.'), documentation_url("4.1.1LDAPAuthentication")%></p>
-  <p>
-    <%= _("Foreman can use External service for user information and authentication.") %><br>
-  </p>
-  <p><%= link_to _('Learn more about External authentication in the documentation.'), documentation_url("5.7ExternalAuthentication") %></p>
-  <div class="blank-slate-pf-main-action">
-    <%= new_link(_("Create LDAP Authentication Source"), :class => 'btn-lg',  :controller => :auth_source_ldaps, :action => :new) %>
-  </div>
-</div>
\ No newline at end of file
+
+<%  content_for(:title, _("Authentication Sources")) %> 
+ 
+<%= react_component('WelcomeAuthSource') %> 
+
+
```

Comment:
Looks like an extra blank line here, and there are several other cases where you added or removed blank lines. 
While not crucial, it's good to avoid too many whitespace changes and try to keep the spacing consistent. Some IDEs even have options to ensure that all extra whitespace is cleaned up before saving automatically.
In ruby files the ruby linter (rubocop) will yell at you in such cases, but we don't lint `.erb` files.